### ISSUE TITLE
Add images to the Tour

### ISSUE THREAD

**Issue by wilzbach** [stonemaster/dlang-tour#244]
I personally love images & I think there are many studies that show that this is a general fact (visual learning is a lot easier for us humans).

Anyway there are a couple of ideas how we could add images to the DLang Tour

1) Add brand images like the DMan

![image](https://cloud.githubusercontent.com/assets/4370550/15980971/049b610e-2f70-11e6-9904-4d7b2b819941.png)

Source: https://github.com/dlang/dlang.org/blob/master/images/d3.png

There are more those lovely D mascot drawing, like this one for Assembler:

![image](https://cloud.githubusercontent.com/assets/4370550/15981017/754b2ed4-2f70-11e6-9beb-7f05da9607e0.png)

https://github.com/dlang/dlang.org/blob/master/images/d5.gif

2) Add more icons (e.g. FontAwesome)

I am not sure yet how we could do that, but it's worthwhile to think about it

3) Add related images (__most important__)

E.g. for Caesor encryption we could show an image like this:

![image](https://cloud.githubusercontent.com/assets/4370550/15981044/c8d49388-2f70-11e6-9b82-56a24a4ffa3b.png)

4) Your ideas

...

**Comment by wilzbach** [comment]
meiz_sandwich sent me his collection of D-mans (> 200!).
I uploaded them on [Github](https://github.com/wilzbach/d-mans) and he gave us the permission to use them. So let's prettify the DLang Tour with them ;-)

I opened PRs for a couple of them to [dlang.org](https://github.com/dlang/dlang.org/pulls?q=is%3Aopen+is%3Apr+label%3A%22Needs+Approval%22). I think some of them would fit to the tour as well.

In any case we should try to avoid such such needed "hacks" between the mobile and desktop site as I did for the frontpage. I don't think this will be an issue for upcoming images, because the problem on the frontpage was the bullet lists which we so far don't have on (any?) other page.
